package.json

```diff
@@ -1,46 +1,46 @@
 {
 	"name": "sveltesociety.dev",
-	"version": "0.0.1",
+	"repository": "https://github.com/svelte-society/sveltesociety.dev.git",
+	"packageManager": "pnpm@8.6.6",
 	"type": "module",
 	"scripts": {
 		"dev": "vite dev",
 		"start": "vite dev --open",
 		"build": "vite build",
 		"preview": "vite preview",
+		"check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+		"test": "vitest",
 		"lint": "prettier --check ./**/*.{js,ts,css,md,svelte,html,json} && eslint --ignore-path .gitignore .",
 		"format": "prettier --write ./**/*.{js,ts,css,md,svelte,html,json}",
-		"test": "jest",
 		"prepare": "husky install"
 	},
 	"devDependencies": {
-		"@macfja/svelte-persistent-store": "1.2.0",
-		"@sveltejs/adapter-static": "^1.0.0",
-		"@sveltejs/kit": "^1.0.0",
+		"@macfja/svelte-persistent-store": "2.3.1",
+		"@sveltejs/adapter-static": "^2.0.2",
+		"@sveltejs/kit": "^1.21.0",
 		"@types/itemsjs": "^2.1.0",
-		"@types/jest": "^29.4.0",
-		"@typescript-eslint/eslint-plugin": "^4.19.0",
-		"@typescript-eslint/parser": "^4.19.0",
-		"eslint": "^7.22.0",
+		"@typescript-eslint/eslint-plugin": "^5.61.0",
+		"@typescript-eslint/parser": "^5.61.0",
+		"eslint": "^8.44.0",
 		"eslint-config-prettier": "^8.1.0",
-		"eslint-plugin-svelte3": "^3.2.0",
+		"eslint-plugin-svelte": "^2.32.2",
 		"graphql-request": "^3.5.0",
 		"highlight.js": "^11.6.0",
 		"husky": "^7.0.2",
 		"itemsjs": "^2.1.20",
-		"jest": "^29.4.1",
-		"lint-staged": "^11.1.2",
-		"mdsvex": "^0.9.3",
-		"prettier": "~2.2.1",
+		"lint-staged": "^13.2.3",
+		"mdsvex": "^0.11.0",
+		"prettier": "^2.8.8",
 		"prettier-plugin-svelte": "^2.2.0",
 		"rehype-slug": "^5.1.0",
-		"svelte": "^3.54.0",
-		"svelte-highlight": "^5.2.0",
-		"svelte-preprocess": "^4.10.2",
+		"svelte": "^4.0.3",
+		"svelte-check": "^3.4.4",
+		"svelte-highlight": "^7.3.0",
 		"svelte-select": "^4.4.0",
-		"ts-jest": "^29.0.5",
 		"tslib": "^2.0.0",
-		"typescript": "^4.4.2",
-		"vite": "^4.0.0"
+		"typescript": "^5.1.6",
+		"vite": "^4.0.0",
```

Comment:
you potentially could upgrade to Vite 4.4.2 as well

Reply:
Done!